composer.json

```
@@ -140,6 +140,7 @@
         "open-y-subprojects/openy_content_core": "^1.1",
         "open-y-subprojects/common_scss": "dev-main",
         "ycloudyusa/y_lb": "^1",
+         "ycloudyusa/y_lb_demo_content": "^1",
```

there is no version yet, let's do like below
```suggestion
        "ycloudyusa/y_lb_demo_content": "^1 || dev-main",
```